src/routes/api/api-resync-post.ts

```diff
@@ -24,7 +40,14 @@ export const ApiResyncPost = async (req: Request, res: Response): Promise<void>
 	const targetTasks = req.body.targetTasks as TaskType[];
 
 	if (!statusTypes && !installationIds && !limit && !inactiveForSeconds) {
-		res.status(400).send("please provide at least one of the filter parameters!");
+		res.status(400).send("Please provide at least one of the filter parameters!");
+		return;
+	}
+
+	const existingInstallationIds = await getIdsForExistingInstallations(installationIds, req.log);
+
+	if (!existingInstallationIds.length) {
+		res.status(400).json("No installations exist for provided IDs");
```

Comment:
Opted to return a 400 instead of a 404 as the Jira ticket recommended - figured this falls more into the category of a bad request than the requested web page not being available.